releasenotes/notes/add-counts-class-7c75bd94d12a161a.yaml

```diff
@@ -0,0 +1,14 @@
+---
+features:
+  - |
+    A new class, :class:`qiskit.result.Counts` has been added. This class
+    is a subclass of ``dict`` and . It is created by passing a dictionary
```

Comment:
incomplete sentence

Reply:
Fixed in: https://github.com/Qiskit/qiskit-terra/pull/4501/commits/3a07fc7d8c2aa83cbc8ee67d7daf01a3be764e8a